ruby_review_week_four.rb

```diff
@@ -2,36 +2,131 @@
 
 # 1. What is a variable?
 
+# varible is a place holder with name to store data.
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+# .each
+
 # 3. In Ruby, what are booleans?
 
+# true and false.
+
 # 4. How do classes relate to objects?
 
+# objects are made out of classes. Class are the general idea of a thing and object is a thing.
+
 # 5. What is an instance variable?
 
+# variables that belong to object are called instance variables.
+
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
+# computer
+
+# def initialize (name, brand)
+#   @name = name
+#   @brand = brand
+# end
+
+# def turn_the_computer_on 
+# end 
+
+
+
 # 7. What is handy about using gems in Ruby?
 
+# gems are a piece of code written by other programmers doing a purticular function which you can use in your program to implement that particual task.
+
 # 8. Define a class called fish, and leave it empty.
 
+# def Fish
+# end
+
 # 9. How would you use that class to make three fish objects?
 
-# 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
+# object = Fish.new
+
+# 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.''
+
+# class Fish
+# attr_accessor:name, :species
+
+# def initialize
+#   @name = name
+#   @species = species
+# end
+
+# end
+
 
 # 11. 
 
 # a. Create one fish object and store it in a variable, and 
 
+# fish_object = Fish.new
+
+
+
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+# fish_object.name=("Fred")
```

Comment:
great work for all of these answers

for the `.name` one you could have just done 

```rb
fish_object.name = "Fred"
```